### PR TITLE
Add episode Confirmation Screen

### LINKED ISSUE #30: Episode Confirmation Screen
After the users have finished inputting all of their episodes for the day, they need to confirm what they inputted. However, people make mistakes, so they should be able to edit the episodes they already have. 

Your task if you choose to accept it: Implement the episode confirmation screen from the Figma designs that show all of the submitted episode information, and allow the user to edit them.

### PR DESCRIPTION
**Pull request by jrosen081**
(There are weird issues with time, ignore them for now, we will be switching our representation of time later)

### CODE REVIEW

File: screens/EpisodeInputFlow/EpisodeEditScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+};
+
+interface TimeProps {
+  time: Date | undefined;
+  setTime: (time: Date | undefined) => void;
+  label: string;
+  style: StyleProp<ViewStyle>;
+}
+
+function TimePicker({ time, setTime, label, style }: TimeProps) {
```
Comment (dfarooq610) [inline screens/EpisodeInputFlow/EpisodeEditScreen.tsx]: We have a time picker in the episode input screen with time props and whatnot, we should probably reuse it here.

File: types.tsx [hunk 2]
```diff
@@ -6,7 +6,8 @@ export type RootStackParamList = {
 export type EpisodeInputStackParamList = {
   EpisodeInput: undefined;
   EpisodeDisplay: undefined;
-  CameraFlow: undefined;
+  EpisodeConfirmation: undefined;
+  EpisodeEdit: undefined;
 };
```
Comment (dfarooq610) [inline types.tsx:11]: this is a typescript thing: but what if this was an enum? Or its kinda just weird how there are a bunch of undefined. Maybe this is a problem for later?

**Review by dfarooq610 — APPROVED** [review]
PRO MOBILE DEV

File: types.tsx (hunk 2 above)
Comment (jrosen081) [reply types.tsx:11]: The interesting thing about this is that this is a React Navigation thing, and needs to be like this. Technically `undefined` represents the params passed to it (even though we don't actually use it sorry not sorry).

**Review by zefwang — APPROVED** [review]
Lettuce, Gouda, Tomatoes, and Mayo, LGTM.

File: screens/EpisodeInputFlow/EpisodeInputCommonStyles.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    fontStyle: "normal",
+    fontSize: 18,
+    lineHeight: 22,
+    marginVertical: "2%",
+  },
+  greyText: {
+    color: "#9c9c9c",
+  },
+  inputContent: {
+    // flex: 2,
```
Comment (jrosen081) [inline screens/EpisodeInputFlow/EpisodeInputCommonStyles.tsx]: Can we delete this?

**Pull request merged**